quill-core/src/main/scala/io/getquill/ast/AstShow.scala

```diff
@@ -29,6 +31,14 @@ object AstShow {
     case Dynamic(tree) => tree.toString
   }
 
+  implicit val blockShow: Show[Block] = Show[Block] {
+    case Block(statements) => statements.map(_.show).mkString(" ")
```

Comment:
I think blocks should be printed inside brackets to properly show nested scopes.